ambry-router/src/main/java/com.github.ambry.router/GetManager.java

```diff
@@ -128,7 +128,7 @@ void submitGetBlobOperation(String blobId, GetBlobOptionsInternal options, Callb
       }
       getOperations.add(getOperation);
     } catch (RouterException e) {
-      routerMetrics.onGetBlobError(e, options);
+      routerMetrics.onGetBlobError(e, options, false);
```

Comment:
it seems like the right thing to do is to convert the string to `BlobId` early in the router stack.

Reply:
sure. will take it up as a separate commit